src/server.js

```diff
@@ -1,9 +1,19 @@
 require('dotenv').config();
+
 const express = require('express');
+const cors = require('cors');
 
 const PORT = process.env.PORT || 3000;
```

Comment:
Use a constants file to have a central repository of exported constants. 

Read this. [https://github.com/nodejs/node/issues/3104](url)